packages/big-design/src/components/Input/Input.tsx

```diff
@@ -24,6 +27,7 @@ export interface Props {
   iconRight?: React.ReactNode;
   label?: React.ReactChild;
   labelId?: string;
+  tooltip?: string;
```

Comment:
- Should we allow to set `placement` for the tooltip? 
- Should we allow to set `trigger` for the tooltip?

According to all the above questions, should it be `string` or `TooltipProps & { text: string }`? 😅

Reply:
@eugene-polev Should we allow the possibility to use any icon instead of a hardcoded `question mark icon`?

Reply:
Agree with @golcinho, we need to know figure out how configurable we want this to be.

Reply:
Looks like here we had almost the same case with a tooltip inside `th`: https://bigcommerce.github.io/big-design/table?props=columns

@vdm-kr Could you please confirm that the tooltip placement should be `right`? or should it be configurable?

Reply:
Just keep the question mark for the icon. 
The default position should be on the right. If possible, it should be automatically adjusted to left/top/bottom when it doesn't fit the screen in the default position.